testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -71,6 +72,7 @@ pub const PATH_BURN_BLOCK_SUBMIT: &str = "new_burn_block";
 pub const PATH_BLOCK_PROCESSED: &str = "new_block";
 pub const PATH_ATTACHMENT_PROCESSED: &str = "attachments/new";
 pub const PATH_PROPOSAL_RESPONSE: &str = "proposal_response";
+pub const PATH_POX_ANCHOR: &str = "new_pox_set";
```

Comment:
I see you can subscribe to new stacker signer sets (and not saying in this PR as this should get merged and this is orthogonal) but do ou think we should add an RPC endpoint for the current pox_set? We will need to retrieve them from the node at signer spin up so we can determine the signers and their vote shares.